.circleci/config.yml

```diff
@@ -43,6 +43,7 @@ jobs:
           working_directory: docs
           environment:
             JAVA_HOME: /usr/lib/jvm/default-java
+            MLFLOW_DISABLE_SYSTEM_METRICS_LOGGING: "True"
```

Comment:
yea, I just updated the code to reflect our discussion.

Thinking about the name - I am keeping the name "MLFLOW_DISABLE_SYSTEM_METRICS_LOGGING" because in the future we will switch to default logging, in which case "MLFLOW_DISABLE_SYSTEM_METRICS_LOGGING" is more clear. Using the same name here would not cause us backward compatibility issues.

Reply:
Yea, I have updated the code to reflect the discussion.

Reply:
I think the env var name should be `MLFLOW_ENABLE_SYSTEM_METRICS_LOGGING`

Reply:
sorry, forgot to update this file, changed